example/ios/Podfile

```diff
@@ -24,3 +24,16 @@ target 'CrispChatSdkExample' do
     react_native_post_install(installer)
   end
 end
+
```

Comment:
No need for this now. I have upgraded master to use the latest version of RN(0.64.1) so all your flipper issues should be fixed